src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/CVService/CVObjectDetectionRequest.cs

```diff
@@ -81,7 +81,7 @@ public void WriteToStringBuilder(StringBuilder stringBuilder)
             if (imageQuery != null)
```

Comment:
these != null checks don't really need to exist, just let the StringJsonConverter write out a `null` value for the json field